### PR TITLE
Merge toe to development

### PR DESCRIPTION
**Pull request by Blasco9**
In this branch, we created the basic HTML structure, added some styles to the page, and built the game separating the logic, board, UI, players, and game objects into modules and factory functions.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Blasco9 and @Ashah15 
### You have done well so far :clap: However, you have a few issues to fix before the project can be approved.
- [ ] Allow players to enter their names before they start playing instead of player one and player two
- [ ] Separate the app logic from dom manipulation
### Make the changes and submit for another review. Good luck.
- Happy coding!

### Important
- You can save players winnings in a localstorage and display results, cumulatively

File: main.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
+const playersArray = [];
```
Comment (wathigo) [inline main.js]: **REQUIRED.**
* It doesn't use global code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from Odin)

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (wathigo) [inline .vscode/settings.json]: **REQUIRED**
*  Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`, `.vscode`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from Pathwright)

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required.
Great job!
The game executes as expected.
Let's make some changes to make the project even better.

**REQUIRED**
* See the comments for pointers :point_down: 

**OPTIONAL**
* Let's arrange the project into different files.
Doing this will improve code readability, which plays a critical role in maintaining the project.

**After making the changes, submit another request for review.**

**Happy Hacking :computer: 
Simon Wathigo.**

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED 

Hi @Blasco9 @Ashah15 good job working on this, the changes looks solid and everything works fine, **this looks good to me now**, please merge and move on **This is approved**

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Pull request merged**